CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * BREAKING: Iterate share links component (PR #316)
 * Add image card component (PR #322)
 * Add notice component (PR #346)
+* Notice title containing markup option (PR #352)
```

Comment:
Not a blocker, but I don't think this needs a changelog entry as it doesn't add/change functionality.